pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -473,19 +526,18 @@ message ContainerConfig {
     // container logs are under active discussion in
     // https://issues.k8s.io/24677. There *may* be future change of direction
     // for logging as the discussion carries on.
-    optional string log_path = 12;
-    // The hash of container config
+    optional string log_path = 10;
 
     // Variables for interactive containers, these have very specialized
     // use-cases (e.g. debugging).
     // TODO: Determine if we need to continue supporting these fields that are
     // part of Kubernetes's Container Spec.
-    optional bool stdin = 13;
-    optional bool stdin_once = 14;
-    optional bool tty = 15;
+    optional bool stdin = 11;
+    optional bool stdin_once = 12;
+    optional bool tty = 13;
 
     // Linux contains configuration specific to Linux containers.
-    optional LinuxContainerConfig linux = 16;
+    optional LinuxContainerConfig linux = 14;
```

Comment:
exactly, once released, these numbers shouldn't be changed. we can arrange these fields before release to make it looking comfortable.